.gitignore

```diff
@@ -15,3 +15,4 @@ Cargo.lock
 
 # Generated by dogfood
 /target_recur/
+.*.sw*
```

Comment:
Haha, that's for vim swapfiles. Can take it out if needed.

-yati
On Dec 23, 2015 5:44 PM, "llogiq" <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/Manishearth/rust-clippy/pull/506#discussion_r48360113>
> :
>
> > @@ -15,3 +15,4 @@ Cargo.lock
> >
> >  # Generated by dogfood
> >  /target_recur/
> > +.*.sw*
>
> What is this for?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Manishearth/rust-clippy/pull/506/files#r48360113>.
>